Portfolio Finish mobile version PR

diarisdiakite opened the pull request:
In this PR I:
- Added the recent works section with a header and a projects class with a display of grid
- Added 5 different projects with a display flex
- Added the image placeholder to the project class
- Added an _ul_ tag with the class project-built-with
- Added the button see project
- Added a competences section

diarisdiakite commented:
In this Milestone, I:
- Fixed placeholder issue for projects
- Fixed text centering issues
- Added buttons interactions

mahbubul14 commented on index.html:
```diff
@@ -55,6 +54,227 @@ <h2 class="green">I’m a Full Stack developer from Microverse</h2>
+    <img src="images/buttons/Disabled.png" alt="">
+  </button>
+
+  <section class="recent-work">
+    <header>
+      <h2 class="green">
+        My recent work
+      </h2>
+      <hr class="green">
+    </header>
+
+    <div class="projects">
+
+      <!-- Project 1 -->
+
+      <div class="project">
+        <div class="project-image">
+          <img src="" alt="" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Setup and mobile version skeleton
+          </h3>
+        </div>
+        <ul class="project-built-with">
+          <li><a href="">Ruby on Rails</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">Javascript</a></li>
+          <li><a href="">HTML</a></li>
+        </ul>
+
+        <div class="project-buttons">
+          <a href="">See project</a>
+        </div>
+        
+      </div>
```
- [ ] Kindly try to match your design to the Figma design. The section has the margins (top, bottom, left, right) defined in the design. The section uses the font and colors (background and font) defined for the template. :+1: 
 
Your design             |  Figma Design
:-------------------------:|:-------------------------:
![yourdesign_work](https://user-images.githubusercontent.com/70483835/227732183-ec40b9e4-e020-4651-adae-7f4b12807dd6.png) | ![figma_work](https://user-images.githubusercontent.com/70483835/227732181-9bcfe7a0-28ec-44b6-8878-f361da517230.png)

mahbubul14 commented on index.html:
```diff
@@ -55,6 +54,227 @@ <h2 class="green">I’m a Full Stack developer from Microverse</h2>
+        </ul>
+    
+      </div>
+
+
+      <div class="frameworks">
+        <div class="competences-image">
+          <img src="images/Icons/frameworks.png" alt="languages" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Frameworks
+          </h3>
+        </div>
+        <ul class="list-competences">
+          <li class="competences-item"><a href="">React.js</a></li>
+          <li class="competences-item"><a href="">Ruby on Rails</a></li>
+          <li class="competences-item"><a href="">RSpec</a></li>
+          <li class="competences-item"><a href="">Capybara</a></li>
+          <li class="competences-item"><a href="">Selenuim</a></li>
+        </ul>
+    
+      </div>
+
+      <div class="skills">
+        <div class="competences-image">
+          <img src="images/Icons/skills.png" alt="languages" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Skills
+          </h3>
+        </div>
+        <ul class="list-competences">
+          <li><a href="">Database management</a></li>
+          <li><a href="">version control</a></li>
+          <li><a href="">CLI</a></li>
+          <li><a href="">Web development</a></li>
+          <li><a href="">API design</a></li>
+        </ul>
```
- [ ] Kindly try to match your design to the Figma design. The section has the margins (top, bottom, left, right) defined in the design. The section uses the font and colors (background and font) defined for the template. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![yourdesign_myself](https://user-images.githubusercontent.com/70483835/227732520-944fd1de-bef4-4266-b5fc-ac0d4e985ecd.png) | ![figma_myself](https://user-images.githubusercontent.com/70483835/227732516-49f176a5-e9cd-47bb-add1-5cbd04bc1329.png)

mahbubul14 reviewed (changes requested):
Hi @diarisdiakite,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

mckent05 commented on index.html:
```diff
@@ -55,6 +54,228 @@ <h2 class="green">I’m a Full Stack developer from Microverse</h2>
+    
+    <div class="competences">
+      
+      <div class="languages">
+        <div class="competences-image">
+          <img src="images/Icons/languages.png" alt="languages" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Languages
+          </h3>
+        </div>
+        <ul class="list-competences">
+          <li><a href="">Javascript</a></li>
+          <li><a href="">Ruby</a></li>
+          <li><a href="">HTML</a></li>
+          <li><a href="">CSS</a></li>
+        </ul>
```
- Just as the last reviewer mentioned, please ensure that your skills and technology cards are styled precisely as in the Figma template. Please fix this 🙏 

Your project:
- Please add a border-radius to the list of technologies 🙏 
<img width="296" alt="qergqerwg" src="https://user-images.githubusercontent.com/73607512/227745334-cb706873-dbfe-4a00-92dc-00e58203d177.PNG">

Design template:
![zsas](https://user-images.githubusercontent.com/73607512/227745375-f89a173d-84cc-498a-842c-5a833e775029.png)

diarisdiakite replied:
Done @mckent05 @mahbubul14

mckent05 commented on index.html:
```diff
@@ -55,6 +54,228 @@ <h2 class="green">I’m a Full Stack developer from Microverse</h2>
+            Desktop version
+          </h3>
+        </div>
+        <ul class="project-built-with">
+          <li><a href="">Ruby on Rails</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">Javascript</a></li>
+          <li><a href="">HTML</a></li>
+        </ul>
```
- Just as the last reviewer mentioned, please ensure that this section is styled precisely as in the Figma template. Please fix this 🙏

Your project:
- Please add a border-radius and ensure they are styled as in the design template 👍 👍 
<img width="273" alt="tetweweg" src="https://user-images.githubusercontent.com/73607512/227745477-048b96fb-b78c-43f6-abd9-cfa25a7e40d4.PNG">

diarisdiakite replied:
Done @mckent05 @mahbubul14

mckent05 reviewed (changes requested):
Hi @diarisdiakite, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![almost there](https://user-images.githubusercontent.com/73607512/207265056-d056ea78-77c5-4b66-802e-694a5ed4ecf3.gif)

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Your portfolio is gradually looking nice, well-done 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

diarisdiakite commented:
In this milestone, I:
- Increased padding of the technology competences lists elements
- Added border radius to the 2 lists elements
- Changed the PR's name to reflect both sections

AbayomiOlaoye commented on css/styles.css:
```diff
@@ -4,11 +4,64 @@
+  margin: 3% auto 30% auto;
+  width: max-content;
+}
+
+.project-buttons:hover {
+  background-color: black;
+}
+
+.project-buttons:enabled {
+  background-color: white;
+}
+
+.see-resume-button:hover {
+  background-color: black;
+}
+
+.see-resume-button:enabled {
+  background-color: white;
 }
```
- [ ] [OPTIONAL] Great efforts there, @diarisdiakite :clap: however, this `background` value of `black` is not conforming with the requirements on Figma. Kindly check, using the screenshot below as guide to get the correct values onFigma.
![image](https://user-images.githubusercontent.com/99715640/227749510-e14cb0b2-3c8b-43c3-987f-2f401f01f7d8.png)

diarisdiakite replied:
Well received, that will be corrected @AbayomiOlaoye

AbayomiOlaoye reviewed (approved):
Hi @diarisdiakite :clap:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![applause](https://user-images.githubusercontent.com/99715640/213818870-485b6c36-1d87-4b93-9a0c-ce2ae86d9a9d.gif)

## Highlights
- Great implementation of reviewer's suggestions :heavy_check_mark: 
- Good UI implementation :heavy_check_mark: 
- Linter checks are passing :heavy_check_mark: 
- Professional ReadMe documentation :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
